includes/Message.php

```diff
@@ -569,7 +569,8 @@ protected function parseText( $string ) {
 		// Wikia change - start
 		// MessageCache::parse can return a string (PLATFORM-807)
 		$parserOutput = MessageCache::singleton()->parse( $string, $this->title, /*linestart*/true, $this->interface, $this->language );
-		Wikia\Util\Assert::true( $parserOutput instanceof ParserOutput, 'Message::parseText: parse() did not return ParserOutput' );
+		Wikia\Util\Assert::true( $parserOutput instanceof ParserOutput, 
+			'Message::parseText: parse() did not return ParserOutput, got ' . wfGetValueExcerpt($parserOutput) );
```

Comment:
I'd add `wfGetValueExcerpt` as a key in an context array that can be passed as a third argument to `Assert::true` method. This way we'll get the consistent message (one ticker created by Jira Reporter), but have enough data for debugging.